hi-ml-cpath/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -231,7 +231,7 @@ def _mock_get_encoder(  # type: ignore
                 TilesDataset.SLIDE_ID_COLUMN: [str(slide_idx)] * bag_size,
                 TilesDataset.TILE_ID_COLUMN: [f"{slide_idx}-{tile_idx}" for tile_idx in range(bag_size)],
                 TilesDataset.IMAGE_COLUMN: rand(bag_size, *input_dim),
-                TilesDataset.LABEL_COLUMN: bag_label.expand(bag_size),
+                LABEL_COLUMN: bag_label.expand(bag_size),
```

Comment:
The need for this change highlights that we should eventually consider switching the pipeline to use standardised `TileKey`s.

Reply:
Definitely, we should keep it in mind for future refinements of the pipeline.